.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Build and run the Go binary from the repo
+# REQUIRES: Go v1.16+ installed
+- id: terraform-docs-go
```

Comment:
`s/terraform-docs-go/terraform-docs/` to match the example yaml down below.

Reply:
I intentionally left this one. I think if you're going to offer multiple hooks that do the same thing via different platforms, then there needs to be a clear distinction that the consumer needs to choose one.

I also think making the `id: terraform-docs` implies this is the reasonable default, which we've established it's not for the Go-binary option because most people don't have v1.16 or higher installed. At best, I'd make the default the Docker option right now, but I think that's inconsistent with most other pre-commit hooks I've seen, hence including the suffix on all three.

Reply:
Fair point, then can you update example to be using go hook. Because the example as it stands is broken.